sopel/modules/currency.py

```diff
@@ -30,9 +29,12 @@
     (([a-zA-Z]{3}$)|([a-zA-Z]{3})\s)+$                          # one or more 3-letter currency code
 ''', re.VERBOSE)
 LOGGER = get_logger(__name__)
+UNSUPPORTED_CURRENCY = "Sorry, {} isn't currently supported."
+GENERIC_SORRY = "Sorry, I didn't understand the input."
```

Comment:
I should have suggested a name for this constant, haha. Maybe `UNRECOGNIZED_INPUT`?